Gemfile

```diff
@@ -22,7 +22,7 @@ gem "rswag-api"
 gem "rswag-ui"
 gem "sentry-rails"
 gem "sentry-ruby"
-gem "sidekiq"
+gem "sidekiq", "~> 7.0"
```

Comment:
Is there a reason we should be specifying this constraint?

Dependabot doesn't pay attention to them anyway, and we rarely update gems manually.

Reply:
pessimistic versioning?! i guess we could get rid of. what do others think?

Reply:
Feel free to merge away, I just wondered what the team thought